### PR TITLE
Elementary comparisons of the declared type of term members.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: tasty-mima/src/main/scala/tastymima/TypeTranslator.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  def translateType(oldType: Type): Type =
+    val alreadyTranslated = translatedRecTypes.get(oldType)
+    if alreadyTranslated != null then alreadyTranslated
+    else
+      oldType match
+        case oldType: NamedType =>
+          oldType.prefix match
+            case oldPrefix: Type =>
+              val translatedPrefix = translateType(oldPrefix)
+              NamedType(translatedPrefix, oldType.name)(using newCtx)
```
Comment (bishabosha) [inline tasty-mima/src/main/scala/tastymima/TypeTranslator.scala:21]: what if it's a `TermRef` to a method with overloads?

Comment (sjrd) [reply tasty-mima/src/main/scala/tastymima/TypeTranslator.scala:21]: It cannot be. A method with parameters cannot appear in a path.

File: test-lib-v2/src/main/scala/testlib/membertypechanges/MemberTypeChanges.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package testlib.membertypechanges
+
+/* tasty-query does not correctly deal with APPLIEDtpt for polymorphic type
```
Comment (bishabosha) [inline test-lib-v2/src/main/scala/testlib/membertypechanges/MemberTypeChanges.scala:3]: is there an issue for this?

Comment (sjrd) [reply test-lib-v2/src/main/scala/testlib/membertypechanges/MemberTypeChanges.scala:3]: Not yet. I need to isolate it as a minimized issue for tasty-query.

**Review by bishabosha — APPROVED** [review]

File: tasty-mima/src/main/scala/tastymima/TypeTranslator.scala (hunk 1 above)
Comment (bishabosha) [reply tasty-mima/src/main/scala/tastymima/TypeTranslator.scala:21]: I meant `oldType`, not `oldPrefix`, will `translateType` only be used for signatures?

**Pull request merged**